src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = sourceString
+    .split(';')
```

Comment:
[CODE PRACTICE] - While using `split` on the source string is correct, you could directly chain the `map` method without storing the intermediate result in a variable, as it would make the code more concise and readable.